### PR TITLE
Styles css after

### PR DESCRIPTION
**Pull request by sampler36**
##  What does this PR do

Fix Css and  Routing Pages to correct States
##  What are the relevant Trello board stories
https://trello.com/c/WVYO2niI

##  Questions (If any):

### CODE REVIEW

**Comment by sampler36** [comment]
[Route About and Contact to non Auth, Style neccessary Css and fix minor bugs](https://trello.com/c/WVYO2niI/134-route-about-and-contact-to-non-auth-style-neccessary-css-and-fix-minor-bugs)

File: workout-tracker/src/views/Settings/Settings.js [hunk 1]
```diff
@@ -452,7 +449,7 @@ class Settings extends React.Component {
                   </li>
                 </ul>
               </div>
-              <button className="update-button" onClick={this.startUpdate}>
+              <button className="button" onClick={this.startUpdate}>
                 Update
               </button>
             </StyledSettings>
```
[the remark this answers is not in the document]
Comment (hyetigran) [reply workout-tracker/src/views/Settings/Settings.js]: When visiting this page, I had to scroll down all the way to the bottom before I could edit. Should we move the update button to the top?

[the remark this answers is not in the document]
Comment (hyetigran) [reply workout-tracker/src/views/Settings/Settings.js]: The divs are animated. Should we change styling from animation to highlight on hover?
Also, when the cards animate, my expectation is that I can click on them and have some actions take place.

[the remark this answers is not in the document]
Comment (Kellswork) [reply workout-tracker/src/views/Settings/Settings.js]: Looks good the navigation works properly

[the remark this answers is not in the document]
Comment (Kellswork) [reply workout-tracker/src/views/Settings/Settings.js]: Your commit messages though lol

**Pull request merged**